### PR TITLE
Update rdf-mapping.md doc

### PR DESCRIPTION
**Pull request by elizoller**
## Purpose / why

Add documentation for change made in: https://github.com/Islandora/documentation/issues/1581

## What changes were made?
Add documentation

## Verification

Make sure the docs are clear.

## Interested Parties

@Islandora/8-x-committers
@seth-shaw-unlv

### CODE REVIEW

**Review by seth-shaw-unlv — CHANGES_REQUESTED** [review]
It is a bit awkward to transition from a bulleted list to the sub-heading. 

Also, the mapping types runs into the datatype_callback when they are really different creatures.

**Comment by elizoller** [comment]
I separated it into two sub-headings. does that improve readability?

File: docs/islandora/rdf-mapping.md [hunk 1]
```diff
@@ -63,8 +63,26 @@ Below is an example of an RDF mapping as a .yml (YAML) file. It is the current v
 
 - The top level key `types` specifies the `rdf:type` of the resource or content model. `field_model`, a required field of Islandora objects, also gets mapped to `rdf:type` through an arcane back-end process.
 - The top level key `fieldMappings` specifies fields attached to that bundle and their RDF property mappings. One field can be mapped to more than one RDF property. It is a simple flat list.
-- `datatype_callback` : [needs documentation]
-- `mapping_type: rel` : [needs documentation]
+
+#### Mapping Types
+`mapping_type:`: There are several mapping types which are provided out of the box. One of those being `rel` - standing for relationship. The default, or if one is not provided is `property` meaning a property.
```
Comment (manez) [inline docs/islandora/rdf-mapping.md]: "The default, or if one is not provided is `property` meaning a property." I'm having a little bit of trouble parsing this line. Is "one" a relationship? If so, maybe:

 "The default, or if a relationship is not provided, is `property` meaning a property."

Comment (elizoller) [reply docs/islandora/rdf-mapping.md]: pushed another commit - is that any clearer?

**Review by manez — APPROVED** [review]

File: docs/islandora/rdf-mapping.md (hunk 1 above)
Comment (manez) [reply docs/islandora/rdf-mapping.md]: Looks good to me

**Comment by rosiel** [comment]
Hey @seth-shaw-unlv i used Github's "re-request review" because merging was blocked by your requested changes (huh!). Sorry for the multi-ping.

**Review by rosiel — APPROVED** [review]
Looks good to me too.

**Review by seth-shaw-unlv — APPROVED** [review]

**Comment by seth-shaw-unlv** [comment]
Sorry for the delay! I was out yesterday to help the kids with their first day back to school online.

**Pull request merged**